modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/dataset_display/entry.js

```diff
@@ -21,7 +21,7 @@ import ViewsContext, {viewsReducer} from './views/ViewsContext';
 
 const App = ({apiURL, appURL, ...props}) => {
 	const {
-		activeViewSettings: {name: activeViewName, ...activeViewSettings},
+		activeViewSettings: {name: activeViewName, visibleFieldNames = {}},
```

Comment:
Funny that [I commented on this cleverness when it went in](https://github.com/liferay-frontend/liferay-portal/pull/209/files#r466895477). I thought it was quite nice at the time. 😁

Reply:
It'll come back eventually 🤣